testdata/scripts/extract.txt

```diff
@@ -690,7 +696,8 @@ curl -X DELETE \
 | 403    | Returned when the user does not have permission to access the resource.                |
 | 500    | Returned when an unexpected error occured on the server side.                          |
 
-## Retrieve an account
+Retrieve an account
+-------------------
```

Comment:
why it changes from ## to ______?

Reply:
Because I thought it's more "common" way to write headers in markdown, so markdownfmt converts `## foo` to 

```
foo
---
```

(original markdownfmt did that too, just our fork didn't)